apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -10,7 +10,7 @@ import AbuseError from './AbuseError';
 import SendToPhone from './SendToPhone';
 import color from '../../util/color';
 import * as applabConstants from '../../applab/constants';
-import * as gamelabConstants from '@cdo/apps/p5lab/gamelab/constants';
+import * as gamelabConstants from '@cdo/apps/p5lab/constants';
```

Comment:
Worth changing to `import * as p5lab constants` at this point?

Reply:
(Also a general question about this PR. I'm guessing you're planning to leave renaming variables/filenames for later)

Reply:
(and renaming other similar variables/filenames in other places). Nevermind about doing it right now - that would be a huge diff 😄 I definitely understand why you didn't do it as part of this change.